engine/src/main/java/org/terasology/rendering/dag/nodes/DownSampleSceneAndUpdateExposureNode.java

```diff
@@ -105,11 +140,10 @@ public void process() {
 
             downSampleSceneInto1x1pixelsBuffer();
 
-            frameBuffersManager.getCurrentReadbackPBO().copyFromFBO(downSampledScene[0].fboId, 1, 1, GL12.GL_BGRA, GL11.GL_UNSIGNED_BYTE);
+            writeOnlyPBO.copyFromFBO(downSampledScene[0].fboId, 1, 1, GL12.GL_BGRA, GL11.GL_UNSIGNED_BYTE);
+            pixels = readOnlyPBO.readBackPixels();
 
-            frameBuffersManager.swapReadbackPBOs();
-
-            pixels = frameBuffersManager.getCurrentReadbackPBO().readBackPixels();
+            swapPBOs();
```

Comment:
The order of the readback and swapping operations have been reversed. Is this intentional?

Beacause the writeOnlyPBO line copies data into writeOnlyPBO. If the readback comes next it should be reading from writeOnlyPBO. Alternatively the next operation could be the swap and the readback from readOnlyPBO then makes sense.  =?